documentation/source/_templates/page.html

```diff
@@ -0,0 +1,14 @@
+{% extends "!page.html" %}
+
+{% block footer %}
+ <script type="text/javascript">
+    $(document).ready(function() {
+        $(".toggle > *").hide();
+        $(".toggle .header").show();
+        $(".toggle .header").click(function() {
```

Comment:
Perhaps better would be https://pypi.org/project/sphinxcontrib-details-directive/